ambry-replication/src/main/java/com/github/ambry/replication/ReplicationMetrics.java

```diff
@@ -704,6 +711,14 @@ public void updateFastestSlowestReplicaPerCycleTimeDifference(long timeDifferenc
     }
   }
 
+  public void updateRemoteReplicaGroupIdleTime(long idleTime, boolean remoteColo, String datacenter) {
```

Comment:
private

Reply:
This needs to be public as it is in ReplicationMetrics class and we will call it from ReplicaThread